src/org/loklak/harvester/RedirectUnshortener.java

```diff
@@ -6,12 +6,12 @@
  *  modify it under the terms of the GNU Lesser General Public
  *  License as published by the Free Software Foundation; either
  *  version 2.1 of the License, or (at your option) any later version.
- *  
+ *
```

Comment:
These changes are introduced by IDE due to [`.editorconfig`](https://github.com/loklak/loklak_server/blob/development/.editorconfig) file and are made to many other files too.